R/0_documentation.R

```diff
@@ -79,56 +79,78 @@ rd_target_schema <- function(type = "param") {
 
 rd_training_length <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("training_length (`numeric`)\\cr",
+  paste("(`numeric`)\\cr",
         "The number of days that should be included in the training of the model.",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
 rd_start_date <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("start_date (`Date`)\\cr",
+  paste("(`Date`)\\cr",
         "Study period start.",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
-rd_end_date <- function(type = "param") {
+rd_slice_ts <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("end_date (`Date`)\\cr",
-        "Study period end.",
+  paste("(`Date` or `character`)\\cr",
+        "Date to slice the database on (used if source_conn is a database).",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
-rd_slice_ts <- function(type = "param") {
+rd_end_date <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("slice_ts (`Date` or `character`)\\cr",
-        "Date to slice the database on (used if source_conn is a database).",
+  paste("(`Date`)\\cr",
+        "Study period end.",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
-rd_.data <- function(type = "param") { # nolint: object_name_linter
+rd_.data <- function(type = "param") {                                                                                  # nolint: object_name_linter
   checkmate::assert_choice(type, c("param", "field"))
   paste(".data\\cr",
-        "The data object to perform the operation on",
+        "The data object to perform the operation on.",
```

Comment:
A preposition is a bad word to end a sentence on:

```suggestion
        "The data object on which to perform the operation.",
```